godtools/App/Share/SwiftUI Views/FullScreenVideoView/FullScreenVideoView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  FullScreenVideoView.swift
+//  godtools
+//
+//  Created by Levi Eggert on 1/31/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+struct FullScreenVideoView: View {
+    
+    @State private var videoPlayerState: VideoViewPlayerState = .stopped
+    
+    @ObservedObject var viewModel: FullScreenVideoViewModel
+    
+    var body: some View {
+        
+        GeometryReader { geometry in
+            
+            VStack(alignment: .center, spacing: 0) {
+                
+                Spacer()
+
+                let aspectRatio: CGSize = CGSize(width: 16, height: 9)
+                let videoWidth: CGFloat = geometry.size.width
+                let videoHeight: CGFloat = (videoWidth / aspectRatio.width) * aspectRatio.height
+                
+                VideoView(playerState: $videoPlayerState, frameSize: CGSize(width: videoWidth, height: videoHeight), videoId: viewModel.videoId, videoPlayerParameters: viewModel.videoPlayerParameters, configuration: VideoViewConfiguration(videoContainerBackgroundColor: .black, videoBackgroundColor: .black, loadingViewBackgroundColor: .black, loadingActivityIndicatorColor: .white), videoEndedClosure: {
+                    viewModel.videoEnded()
+                })
+                .frame(width: videoWidth, height: videoHeight)
+                
+                
+                Spacer()
+            }
+        }
+        .onAppear {
+            videoPlayerState = .playing
+        }
+        .onDisappear {
+            videoPlayerState = .paused
+        }
+        .statusBarHidden(true)
+        .background(Color.black)
```

Comment:
I'm seeing a white bar in the safe area of the view.  May need to make this ignore the safe area? 
<img width="240" alt="Screen Shot 2023-02-06 at 2 02 56 PM" src="https://user-images.githubusercontent.com/8647064/217063069-177437e6-1d08-4205-a60f-191ddb6d6f51.png">

Reply:
Hey @rachaelblue good catch.  Instead I will apply the background to the Hosting ViewController this way the safe area will stay intact.  Feels safer to keep the safe area in place in the event any additional UI elements were added to this view at the bottom of the device screen.